### ISSUE TITLE
Failing build checks for xdg-config access

### ISSUE THREAD

**Issue by s96Abrar** [flathub/flatpak-builder-lint#167]
Using `--filesystem=xdg-config` causes failing for build checks. Below is the error given from the flathub buildbot.

```
flatpak run --command=flatpak-builder-lint org.flatpak.Builder --exceptions org.cubocore.CoreAction.yml
 in dir /srv/buildbot/worker/build-x86_64-6/build (timeout 1200 secs)
 watching logfiles {}
 argv: b'flatpak run --command=flatpak-builder-lint org.flatpak.Builder --exceptions org.cubocore.CoreAction.yml'
 using PTY: False
{
    "errors": [
        "finish-args-arbitrary-xdg-config-access"
    ]
}
If you think problems listed above are a false positive, please report it here:
  https://github.com/flathub/flatpak-builder-lint
program finished with exit code 1
elapsedTime=0.638376
```

**Comment by hfiguiere** [comment]
This is all the intent of the linter. It works as advertised.

**Comment by hfiguiere** [comment]
https://docs.flathub.org/docs/for-app-authors/linter/
